2012/10/15 17:18:29 [error] [php] date(): It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected 'America/Caracas' for 'VET/-4.0/no DST' instead (/Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/protected/views/layouts/main.php:51)
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CBaseController.php(96): SiteController->renderInternal()
#1 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/widgets/CContentDecorator.php(77): SiteController->renderFile()
#2 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/widgets/CContentDecorator.php(55): CContentDecorator->decorate()
#3 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/widgets/COutputProcessor.php(45): CContentDecorator->processOutput()
#4 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CBaseController.php(207): CContentDecorator->run()
#5 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CBaseController.php(301): SiteController->endWidget()
#6 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/protected/views/layouts/column1.php(6): SiteController->endContent()
#7 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CBaseController.php(127): require()
#8 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CBaseController.php(96): SiteController->renderInternal()
#9 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CController.php(785): SiteController->renderFile()
#10 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/protected/controllers/SiteController.php(32): SiteController->render()
#11 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/actions/CInlineAction.php(50): SiteController->actionIndex()
#12 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CController.php(309): CInlineAction->runWithParams()
#13 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CController.php(287): SiteController->runAction()
#14 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CController.php(266): SiteController->runActionWithFilters()
#15 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CWebApplication.php(283): SiteController->run()
#16 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController()
#17 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#18 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/index.php(13): CWebApplication->run()
#19 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/cli/commands/ShellCommand.php(80): require()
#20 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/console/CConsoleCommandRunner.php(68): ShellCommand->run()
#21 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/console/CConsoleApplication.php(92): CConsoleCommandRunner->run()
#22 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/base/CApplication.php(162): CConsoleApplication->processRequest()
#23 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/yiic.php(34): CConsoleApplication->run()
#24 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/protected/yiic.php(7): require_once()
#25 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/protected/yiic(4): require_once()
REQUEST_URI=/index.php
2012/10/15 17:18:41 [error] [php] date(): It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected 'America/Caracas' for 'VET/-4.0/no DST' instead (/Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/protected/views/layouts/main.php:51)
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CBaseController.php(96): SiteController->renderInternal()
#1 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/widgets/CContentDecorator.php(77): SiteController->renderFile()
#2 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/widgets/CContentDecorator.php(55): CContentDecorator->decorate()
#3 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/widgets/COutputProcessor.php(45): CContentDecorator->processOutput()
#4 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CBaseController.php(207): CContentDecorator->run()
#5 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CBaseController.php(301): SiteController->endWidget()
#6 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/protected/views/layouts/column1.php(6): SiteController->endContent()
#7 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CBaseController.php(127): require()
#8 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CBaseController.php(96): SiteController->renderInternal()
#9 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CController.php(785): SiteController->renderFile()
#10 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/protected/controllers/SiteController.php(32): SiteController->render()
#11 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/actions/CInlineAction.php(50): SiteController->actionIndex()
#12 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CController.php(309): CInlineAction->runWithParams()
#13 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CController.php(287): SiteController->runAction()
#14 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CController.php(266): SiteController->runActionWithFilters()
#15 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CWebApplication.php(283): SiteController->run()
#16 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController()
#17 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#18 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/index.php(13): CWebApplication->run()
#19 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/cli/commands/ShellCommand.php(80): require()
#20 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/console/CConsoleCommandRunner.php(68): ShellCommand->run()
#21 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/console/CConsoleApplication.php(92): CConsoleCommandRunner->run()
#22 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/base/CApplication.php(162): CConsoleApplication->processRequest()
#23 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/yiic.php(34): CConsoleApplication->run()
#24 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/protected/yiic.php(7): require_once()
#25 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/protected/yiic(4): require_once()
REQUEST_URI=/index.php
2012/10/15 17:22:01 [error] [php] date(): It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected 'America/Caracas' for 'VET/-4.0/no DST' instead (/Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/protected/views/layouts/main.php:51)
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CBaseController.php(96): SiteController->renderInternal()
#1 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/widgets/CContentDecorator.php(77): SiteController->renderFile()
#2 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/widgets/CContentDecorator.php(55): CContentDecorator->decorate()
#3 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/widgets/COutputProcessor.php(45): CContentDecorator->processOutput()
#4 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CBaseController.php(207): CContentDecorator->run()
#5 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CBaseController.php(301): SiteController->endWidget()
#6 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/protected/views/layouts/column1.php(6): SiteController->endContent()
#7 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CBaseController.php(127): require()
#8 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CBaseController.php(96): SiteController->renderInternal()
#9 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CController.php(785): SiteController->renderFile()
#10 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/protected/controllers/SiteController.php(32): SiteController->render()
#11 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/actions/CInlineAction.php(50): SiteController->actionIndex()
#12 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CController.php(309): CInlineAction->runWithParams()
#13 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CController.php(287): SiteController->runAction()
#14 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CController.php(266): SiteController->runActionWithFilters()
#15 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CWebApplication.php(283): SiteController->run()
#16 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController()
#17 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#18 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/index.php(13): CWebApplication->run()
#19 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/cli/commands/ShellCommand.php(80): require()
#20 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/console/CConsoleCommandRunner.php(68): ShellCommand->run()
#21 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/console/CConsoleApplication.php(92): CConsoleCommandRunner->run()
#22 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/base/CApplication.php(162): CConsoleApplication->processRequest()
#23 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/framework/yiic.php(34): CConsoleApplication->run()
#24 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/protected/yiic.php(7): require_once()
#25 /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/protected/yiic(4): require_once()
REQUEST_URI=/index.php
2012/10/18 20:00:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Dropbox".' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php:287
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('Dropbox')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/Dropbox
---
2012/10/18 20:20:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Dropbox".' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php:287
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('Dropbox')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/Dropbox
---
2012/10/18 20:20:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "dropbox".' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php:287
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('dropbox')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/dropbox
---
2012/10/18 20:21:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Dropbox".' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php:287
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('Dropbox')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/Dropbox
---
2012/10/19 13:15:48 [error] [php] Invalid argument supplied for foreach() (/Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/controllers/DropboxController.php:41)
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(287): DropboxController->runAction()
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(266): DropboxController->runActionWithFilters()
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): DropboxController->run()
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#5 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CWebApplication->run()
REQUEST_URI=/proyectoDesarrollo/index.php/dropbox
in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/controllers/DropboxController.php (41)
in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php (13)
2012/10/19 13:17:02 [error] [php] Cannot modify header information - headers already sent by (output started at /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/controllers/DropboxController.php:2) (/Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CHttpRequest.php:763)
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/controllers/DropboxController.php(85): DropboxController->redirect()
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/actions/CInlineAction.php(50): DropboxController->actionAuthorize_step2()
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(309): CInlineAction->runWithParams()
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(287): DropboxController->runAction()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(266): DropboxController->runActionWithFilters()
#5 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): DropboxController->run()
#6 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController()
#7 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#8 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CWebApplication->run()
REQUEST_URI=/proyectoDesarrollo/index.php/dropbox/Authorize_step2?uid=21927996&oauth_token=oor48l5jppfxbcy
in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/controllers/DropboxController.php (85)
in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php (13)
2012/10/19 13:17:48 [error] [php] Cannot modify header information - headers already sent by (output started at /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/controllers/DropboxController.php:80) (/Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CHttpRequest.php:763)
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/controllers/DropboxController.php(84): DropboxController->redirect()
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/actions/CInlineAction.php(50): DropboxController->actionAuthorize_step2()
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(309): CInlineAction->runWithParams()
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(287): DropboxController->runAction()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(266): DropboxController->runActionWithFilters()
#5 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): DropboxController->run()
#6 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController()
#7 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#8 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CWebApplication->run()
REQUEST_URI=/proyectoDesarrollo/index.php/dropbox/Authorize_step2?uid=21927996&oauth_token=oor48l5jppfxbcy
in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/controllers/DropboxController.php (84)
in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php (13)
2012/10/19 13:17:58 [error] [php] Cannot modify header information - headers already sent by (output started at /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/controllers/DropboxController.php:80) (/Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CHttpRequest.php:763)
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/controllers/DropboxController.php(84): DropboxController->redirect()
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/actions/CInlineAction.php(50): DropboxController->actionAuthorize_step2()
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(309): CInlineAction->runWithParams()
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(287): DropboxController->runAction()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(266): DropboxController->runActionWithFilters()
#5 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): DropboxController->run()
#6 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController()
#7 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#8 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CWebApplication->run()
REQUEST_URI=/proyectoDesarrollo/index.php/dropbox/Authorize_step2?uid=21927996&oauth_token=oor48l5jppfxbcy
in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/controllers/DropboxController.php (84)
in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php (13)
2012/10/19 13:22:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Authorize".' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php:287
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('Authorize')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/Authorize
---
2012/10/19 13:39:51 [error] [exception.Dropbox_Exception_NotFound] exception 'Dropbox_Exception_NotFound' with message 'Resource at uri: https://api-content.dropbox.com/1/thumbnails/dropbox/Amigos could not be found. No thumbnail available for the file type 'Amigos'
' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/extensions/dropbox/OAuth/Curl.php:128
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/extensions/dropbox/API.php(267): Dropbox_OAuth_Curl->fetch('https://api-con...', Array)
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/controllers/DropboxController.php(41): Dropbox_API->getThumbnail('/Amigos')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/actions/CInlineAction.php(50): DropboxController->actionIndex()
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#5 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('')
#7 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('dropbox')
#8 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#9 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/dropbox
---
2012/10/19 13:42:21 [error] [exception.Dropbox_Exception_NotFound] exception 'Dropbox_Exception_NotFound' with message 'Resource at uri: https://api-content.dropbox.com/1/thumbnails/dropbox/Angelica-Diego could not be found. No thumbnail available for the file type 'Angelica-Diego'
' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/extensions/dropbox/OAuth/Curl.php:128
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/extensions/dropbox/API.php(267): Dropbox_OAuth_Curl->fetch('https://api-con...', Array)
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/controllers/DropboxController.php(41): Dropbox_API->getThumbnail('/Angelica-Diego')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/actions/CInlineAction.php(50): DropboxController->actionIndex()
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#5 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('')
#7 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('dropbox')
#8 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#9 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/dropbox
---
2012/10/19 21:55:24 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/auth/CAccessControlFilter.php:170
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/auth/CAccessControlFilter.php(133): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(292): CFilterChain->run()
#6 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('delete')
#8 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('libreta/delete')
#9 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#10 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/libreta/delete/1
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/libreta/1
---
2012/10/19 21:57:14 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/auth/CAccessControlFilter.php:170
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/auth/CAccessControlFilter.php(133): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(292): CFilterChain->run()
#6 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('admin')
#8 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('libreta/admin')
#9 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#10 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/libreta/admin
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/libreta
---
2012/10/19 21:57:23 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/auth/CAccessControlFilter.php:170
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/auth/CAccessControlFilter.php(133): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(292): CFilterChain->run()
#6 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('admin')
#8 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('libreta/admin')
#9 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#10 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/libreta/admin
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/libreta/1
---
2012/10/19 22:04:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "registrar".' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('registrar')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('registrar')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/registr...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 22:06:05 [error] [exception.CException] exception 'CException' with message 'Property "Usuario.verifyCode" is not defined.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CComponent.php:131
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/db/ar/CActiveRecord.php(144): CComponent->__get('verifyCode')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/helpers/CHtml.php(2158): CActiveRecord->__get('verifyCode')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/helpers/CHtml.php(1887): CHtml::resolveValue(Object(Usuario), 'verifyCode')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/helpers/CHtml.php(1217): CHtml::activeInputField('text', Object(Usuario), 'verifyCode', Array)
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/widgets/CActiveForm.php(658): CHtml::activeTextField(Object(Usuario), 'verifyCode', Array)
#5 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/views/usuario/registrar.php(84): CActiveForm->textField(Object(Usuario), 'verifyCode')
#6 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CBaseController.php(127): require('/Applications/X...')
#7 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CBaseController.php(96): CBaseController->renderInternal('/Applications/X...', Array, true)
#8 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(870): CBaseController->renderFile('/Applications/X...', Array, true)
#9 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(783): CController->renderPartial('registrar', Array, true)
#10 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/controllers/UsuarioController.php(99): CController->render('registrar', Array)
#11 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/actions/CInlineAction.php(50): UsuarioController->actionRegistrar()
#12 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#13 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#14 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CFilter.php(41): CFilterChain->run()
#15 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(1146): CFilter->filter(Object(CFilterChain))
#16 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#17 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#18 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(292): CFilterChain->run()
#19 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#20 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('registrar')
#21 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/registr...')
#22 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#23 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#24 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 22:08:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "captcha".' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('captcha')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('captcha')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/captcha')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/captcha?v=5081cf7179cb9
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 22:08:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "captcha".' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('captcha')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('captcha')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/captcha')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/captcha?refresh=1&_=1350684535905
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 22:10:51 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/auth/CAccessControlFilter.php:170
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/auth/CAccessControlFilter.php(133): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(292): CFilterChain->run()
#6 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(266): CController->runActionWithFilters(Object(CCaptchaAction), Array)
#7 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('captcha')
#8 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/captcha')
#9 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#10 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/captcha?v=5081cfeb415d0
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 22:10:52 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/auth/CAccessControlFilter.php:170
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/auth/CAccessControlFilter.php(133): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(292): CFilterChain->run()
#6 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(266): CController->runActionWithFilters(Object(CCaptchaAction), Array)
#7 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('captcha')
#8 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/captcha')
#9 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#10 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/captcha?v=5081cfec2cc74
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 22:10:52 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/auth/CAccessControlFilter.php:170
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/auth/CAccessControlFilter.php(133): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(292): CFilterChain->run()
#6 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(266): CController->runActionWithFilters(Object(CCaptchaAction), Array)
#7 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('captcha')
#8 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/captcha')
#9 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#10 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/captcha?v=5081cfec69097
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 22:14:55 [error] [exception.CException] exception 'CException' with message 'Property "Usuario.clave2" is not defined.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CComponent.php:131
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/db/ar/CActiveRecord.php(144): CComponent->__get('clave2')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/helpers/CHtml.php(2158): CActiveRecord->__get('clave2')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/helpers/CHtml.php(1887): CHtml::resolveValue(Object(Usuario), 'clave2')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/helpers/CHtml.php(1351): CHtml::activeInputField('password', Object(Usuario), 'clave2', Array)
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/widgets/CActiveForm.php(688): CHtml::activePasswordField(Object(Usuario), 'clave2', Array)
#5 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/views/usuario/registrar.php(56): CActiveForm->passwordField(Object(Usuario), 'clave2', Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CBaseController.php(127): require('/Applications/X...')
#7 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CBaseController.php(96): CBaseController->renderInternal('/Applications/X...', Array, true)
#8 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(870): CBaseController->renderFile('/Applications/X...', Array, true)
#9 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(783): CController->renderPartial('registrar', Array, true)
#10 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/controllers/UsuarioController.php(114): CController->render('registrar', Array)
#11 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/actions/CInlineAction.php(50): UsuarioController->actionRegistrar()
#12 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#13 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#14 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CFilter.php(41): CFilterChain->run()
#15 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(1146): CFilter->filter(Object(CFilterChain))
#16 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#17 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#18 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(292): CFilterChain->run()
#19 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#20 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('registrar')
#21 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/registr...')
#22 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#23 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#24 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 22:18:26 [warning] [application] Failed to set unsafe attribute "clave2" of "Usuario".
in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/controllers/UsuarioController.php (205)
in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/controllers/UsuarioController.php (103)
in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php (13)
2012/10/19 22:18:38 [warning] [application] Failed to set unsafe attribute "clave2" of "Usuario".
in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/controllers/UsuarioController.php (205)
in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/controllers/UsuarioController.php (103)
in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php (13)
2012/10/19 22:18:41 [warning] [application] Failed to set unsafe attribute "clave2" of "Usuario".
in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/controllers/UsuarioController.php (205)
in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/protected/controllers/UsuarioController.php (103)
in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php (13)
2012/10/19 22:58:38 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/auth/CAccessControlFilter.php:170
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/auth/CAccessControlFilter.php(133): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(292): CFilterChain->run()
#6 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('admin')
#8 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('libreta/admin')
#9 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#10 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/libreta/admin
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/libreta/1
---
2012/10/19 23:16:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "css" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('css')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('css')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/css/lay...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/css/layout.css
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:16:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "css" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('css')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('css')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/css/sty...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/css/style.css
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:16:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "css" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('css')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('css')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/css/res...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/css/reset.css
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:16:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "js" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('js')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('js')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/js/jque...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/js/jquery-1.4.2.js
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:16:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "js" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('js')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('js')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/js/cufo...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/js/cufon-yui.js
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:16:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "js" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('js')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('js')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/js/cufo...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/js/cufon-replace.js
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:16:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "js" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('js')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('js')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/js/Myri...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/js/Myriad_Pro_300.font.js
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:16:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "js" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('js')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('js')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/js/Myri...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/js/Myriad_Pro_400.font.js
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:16:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/images/...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/images/page1_img1.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:16:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/images/...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/images/page1_img2.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:17:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/images/...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/images/page1_img1.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:17:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/images/...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/images/page1_img2.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:19:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/images/...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/images/page1_img2.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:19:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/images/...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/images/page1_img1.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:21:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/images/...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/images/page1_img1.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:21:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/images/...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/images/page1_img2.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:22:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/images/...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/images/page1_img1.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:22:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/images/...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/images/page1_img2.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:22:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/images/...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/images/page1_img2.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:22:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/images/...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/images/page1_img1.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:25:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/images/...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/images/page1_img1.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:25:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/images/...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/images/page1_img2.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:26:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/images/...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/images/page1_img2.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:26:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/images/...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/images/page1_img1.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:29:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/images/...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/images/page1_img2.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:29:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('usuario/images/...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/usuario/images/page1_img1.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/usuario/registrar
---
2012/10/19 23:29:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('site/images/pag...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/site/images/page1_img1.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/site/page?view=about
---
2012/10/19 23:29:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('site/images/pag...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/site/images/page1_img2.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/site/page?view=about
---
2012/10/19 23:29:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('site/images/pag...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/site/images/page1_img1.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/site/contact
---
2012/10/19 23:29:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('site/images/pag...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/site/images/page1_img2.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/site/contact
---
2012/10/19 23:29:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('site/images/pag...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/site/images/page1_img2.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/site/login
---
2012/10/19 23:29:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('site/images/pag...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/site/images/page1_img1.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/site/login
---
2012/10/19 23:30:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('site/images/pag...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/site/images/page1_img1.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/site/login
---
2012/10/19 23:30:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php:484
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CController.php(271): CController->missingAction('images')
#1 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(283): CController->run('images')
#2 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/web/CWebApplication.php(142): CWebApplication->runController('site/images/pag...')
#3 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/proyectoDesarrollo/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/proyectoDesarrollo/index.php/site/images/page1_img2.jpg
HTTP_REFERER=http://localhost/proyectoDesarrollo/index.php/site/login
---
2012/10/22 15:37:24 [error] [application] CocoWidget an error occurs.
in /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/protected/extensions/coco/CocoWidget.php (251)
in /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/protected/extensions/coco/CocoWidget.php (206)
in /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/protected/extensions/coco/CocoWidget.php (190)
2012/10/22 15:39:41 [error] [application] CocoWidget an error occurs.
in /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/protected/extensions/coco/CocoWidget.php (251)
in /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/protected/extensions/coco/CocoWidget.php (206)
in /Applications/XAMPP/xamppfiles/htdocs/ProyectoDesarrollo/protected/extensions/coco/CocoWidget.php (190)
